pyproject.toml

```
@@ -58,10 +58,10 @@ file = 'LICENSE.txt'
 
 [project.optional-dependencies]
 build = [
-    'meson-python>=0.13.0rc0',
+    'git+https://github.com/mesonbuild/meson-python',
```

Perhaps try:

```suggestion
    'meson-python @ git+https://github.com/mesonbuild/meson-python',
```

Safer to specify a hash in addition, though.